R/read_and_prepare.R

```diff
@@ -10,25 +10,25 @@
 #' @param scenario_geography_filter String holding geographic asset locations to
 #'   be considered in analysis.
 #' @param asset_type Type of asset, can be bonds, loans or equity.
+#' @param path String holding path to data.
 #'
 #' @return A list holding prepared project agnostic data.
 read_and_prepare_project_agnostic_data <- function(start_year, end_year, company_exclusion,
-                                                   scenario_geography_filter, asset_type) {
+                                                   scenario_geography_filter, asset_type,
+                                                   path) {
```

Comment:
The argument that provides the data is usually in position 1. This is particularly the case for functions that read data

``` r
readers <- c(read.csv, readr::read_csv)
lapply(readers, \(x) formals(x)[1])
#> [[1]]
#> [[1]]$file
#> 
#> 
#> 
#> [[2]]
#> [[2]]$file
```

<sup>Created on 2021-11-10 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>